data/mainnet/tokens.js

```diff
@@ -150,6 +150,26 @@ module.exports = {
     ],
     cmcRewardTokenSymbols: ['miFARM', 'pGENE'],
   },
+  mUSD: {
+    chain: CHAINS_ID.MATIC_MAINNET,
+    category: VAULT_CATEGORIES_IDS.MSTABLE,
+    isNew: true,
+    logoUrl: './icons/mstable.svg',
+    apyIconUrls: ['./icons/mta.svg', './icons/wmatic.png'],
+    apyTokenSymbols: ['MTA', 'WMATIC'],
+    displayName: 'mStable_mUSD',
+    tokenAddress: addresses.MATIC.V2.mUSD.Underlying,
+    decimals: '18',
+    vaultAddress: addresses.MATIC.V2.mUSD.NewVault,
+    priceFunction: { type: GET_PRICE_TYPES.COINGECKO_CONTRACT, params: [addresses.mUSD] },
+    estimateApyFunctions: [
+      {
+        type: ESTIMATED_APY_TYPES.MSTABLE,
```

Comment:
The reduction coming from the performance fee needs to be added in here. On polygon we take a 8% performance fee on rewards